awx/main/scheduler/task_manager.py

```diff
@@ -539,6 +552,10 @@ def process_pending_tasks(self, pending_tasks):
                     logger.debug("No instance available in group {} to run job {} w/ capacity requirement {}".format(
                                  rampart_group.name, task.log_format, task.task_impact))
             if not found_acceptable_queue:
+                job_explanation = "This job is not ready to start because there is not enough available capacity."
+                if task.job_explanation != job_explanation and task.created < (tz_now() - timedelta(seconds=30)):
+                    task.job_explanation = "This job is not ready to start because there is not enough available capacity."
```

Comment:
nitpick, but `task.job_explanation = job_explanation`

Reply:
good catch thanks. I also realize I need to unset/change this job_explanation when we start the task

Reply:
Yes, good call.  We may want to make sure that when a job goes into `waiting`, we unset this explicitly.